src/Idle.DataAccess/ModelBase.cs

```diff
@@ -0,0 +1,54 @@
+﻿using SQLite;
+using System;
+using System.Collections.Generic;
+using System.Text;
+
+namespace Idle.DataAccess
+{
+    /// <summary>
+    /// Base class for all models
+    /// </summary>
+    public abstract class ModelBase
+    {
+
+        /// <summary>
+        /// Primary Key for SQLite database
+        /// </summary>
+        [AutoIncrement]
```

Comment:
i would comment this out for now till we add the sqlite nuget, so the solution is in a buildable state
also the `using SQlite;`

Reply:
yep.